test/common/common/BUILD

```diff
@@ -69,3 +71,20 @@ envoy_cc_test(
     srcs = ["callback_impl_test.cc"],
     deps = ["//source/common/common:callback_impl_lib"],
 )
+
+envoy_cc_library(
+    name = "utility_speed_test_lib",
+    srcs = ["utility_speed_test.cc"],
+    external_deps = [
+        "abseil_strings",
+        "benchmark",
+    ],
+    deps = [
+        "//source/common/common:utility_lib",
+    ],
+)
+
+envoy_cc_binary(
+    name = "utility_speed_test",
+    deps = ["utility_speed_test_lib"],
+)
```

Comment:
How come the binary/lib split here?

Reply:
envoy_cc_binary does not allow an external lib.  I'll add that as a comment.

Reply:
You could also extend `envoy_cc_binary` to do this, it's only a couple of lines in the `.bzl` if you look at what `envoy_cc_library` does.